src/backend/src/main/kotlin/org/icpclive/admin/Routing.kt

```diff
@@ -98,7 +99,11 @@ fun Route.configureAdminApiRouting() {
 
         route("/advancedJson") {
             get {
-                call.respondFile(Config.cdsSettings.advancedJsonPath.toFile())
+                try {
+                    call.respondFile(Config.cdsSettings.advancedJsonPath.toFile())
+                } catch (e: IOException) {
```

Comment:
It'd say it is better to check on exists() instead of catching an exception. IOException, can occur because of different reasons unrealaed to files (like, failing to write answer to socket)